Add 2025_SkourtaniotiJia_SCaucasus

xiaowenjia opened the pull request:
### PR Checklist for a new package submission

- [x] The package does not exist already in the community archive, also not with a different name.
- [x] The package title in the `POSEIDON.yml` conforms to the general title structure suggested here: `<Year>_<Last name of first author>_<Region, time period or special feature of the paper>`, e.g. `2021_Zegarac_SoutheasternEurope`, `2021_SeguinOrlando_BellBeaker` or `2021_Kivisild_MedievalEstonia`.
- [x] The package is stored in a directory that is named like the package title.

***

- [x] Samples that already have been published previously, and got re-analysed (e.g. re-sequenced) for the now packaged publication, have a modified `Poseidon_ID` of the form `<Original Poseidon_ID>_<Initials of the main author>_<Year>`. Re-analysed versions of `I1685` (Lazaridis et al. 2016) should, for example, be assigned the IDs `I1685_IL22` (Lazaridis et al. 2022) and `I1685_IL25` (Lazaridis et al. 2025).

***

- [x] The package is complete and features the following elements:
  - [x] Genotype data in binary PLINK format (not EIGENSTRAT format).
  - [x] Genotype has been provided by the original authors of the publication describing the data.
  - [x] A `POSEIDON.yml` file with not just the file-referencing fields, but also the following meta-information fields present and filled: `poseidonVersion`, `title`, `description`, `contributor`, `packageVersion`, `lastModified` (see [here](https://github.com/poseidon-framework/poseidon-schema/blob/master/POSEIDON_yml_fields.tsv) for their definition)
  - [x] A reasonably filled `.janno` file (for a list of available fields look [here](https://github.com/poseidon-framework/poseidon-schema/blob/master/janno_columns.tsv) and [here](https://www.poseidon-adna.org/#/janno_details) for more detailed documentation about them).
  - [x] A `.bib` file with the necessary literature references for each sample in the `.janno` file.
- [x] Every file in the submission is correctly referenced in the `POSEIDON.yml` file and there are no additional, supplementary files in the submission that are not documented there.
- [x] Genotype data, `.janno` and `.bib` file are all named after the package title and only differ in the file extension.
- [x] The package version in the `POSEIDON.yml` file is `1.0.0`.
- [x] The `poseidonVersion` of the package in the `POSEIDON.yml` file is set to the latest version of the [Poseidon schema](https://github.com/poseidon-framework/poseidon-schema/releases).
- [x] The `POSEIDON.yml` file contains the corresponding checksums for the fields `genoFile`, `snpFile`, `indFile`, `jannoFile` and `bibFile`.
- [x] There is either no `CHANGELOG` file or one with a single entry for version `1.0.0`.

***

- [x] The `Publication` column in the `.janno` file is filled and the respective `.bib` file has complete entries for the listed mentioned keys.
- [x] The `.janno` file does not include any empty columns or columns only filled with `n/a`.
- [x] The order of columns in the `.janno` file adheres to the standard order as defined in the Poseidon schema [here](https://github.com/poseidon-framework/poseidon-schema/blob/master/janno_columns.tsv).
- [x] The `.janno` and the `.ssf` files are not fully quoted, so they only use single- or double quotes (`"..."`, `'...'`) to enclose text fields where it is strictly necessary (i.e. their entry includes a TAB).

***

- [x] The package passes a validation with `trident validate --fullGeno`.

***

- [ ] Large genotype data files are properly tracked with Git LFS and not directly pushed to the repository. For an instruction on how to set up Git LFS please look [here](https://www.poseidon-adna.org/#/archive_submission_guide?id=submitting-the-package). If you accidentally pushed the files the wrong way you can fix it with `git lfs migrate import --no-rewrite path/to/file.bed` (see [here](https://github.com/git-lfs/git-lfs/blob/main/docs/man/git-lfs-migrate.adoc#import-without-rewriting-history)).

xiaowenjia commented:
@nevrome thank you so much for the quick review! Now I fixed the line endings!

nevrome commented:
OK - this looks ready for review! But I think the paper is not out yet, right? The bibtex entry is still missing a doi. I assume we should wait with merging the package until everything is properly released.

Could you review, @martynamolak? You have offered to do so in the past, so that's why I'm asking you directly. Feel free to decline the request if this comes at a busy time :+1:! The review guide is available [here](https://www.poseidon-adna.org/#/archive_reviewer_guide).

martynamolak commented:
Hey, if noone else signed up for that yet, I can do it, but probably only early next week. If that is fine than count me in.

nevrome commented:
That is excellent! Thank you very much, Martina!

martynamolak commented:
yml file: Shouldn't @xiaowenjia be rather listed as contributor? But up to you of course

bib file: There is supposed to be a month for the publications listed according to the reviewer's documentation. Could you provide these? (I added them in the file myself now)

janno file:
- in Alternative_IDs you use "/" which I'm afraid might be problematic for some processing pipelines. I would suggest changing to "-"; especially since there are already some indivs (eg. "KMR001-NOK002_unresolved_origin") that use "-" rather than "/"; @nevrome or are "/" in Alternative_IDs fine by Poseidon? (Poseidon_ID and Group_Name are free of these)
- Relation_To: when you list more than one indiv here, you should probably separate them using ";" (now you have "," in some cases and ";" in some others); also not sure how that works when you have individuals related to individuals that are not included in this package (eg. "geo005"). Poseidon_IDs are probably not screened for uniqueness across packages so it might be tricky to identify which individual (from which packages) you are referring to; perhaps you could refer to the source package for the individual in the Relation_Note field.
- Relation_Degree: my feeling is that when you list a number of individuals in "Relation_to", each one should have a corresponding "Relation_Degree" value (you have that in most cases but the ATK individuals only have one Relation_Degree despite having many Relation_to listed); there are "thirdTofifth" Relation_Degree values with a note "BREAD", which is probably a mistake as BREADR does not infer such relations
- Relation_Note: change "BREAD" to "BREADR"; why do some individuals have "n/a" in this field (it's probably fine, but is a bit weird)?
- Location: fix the "Samtskheâ€“Javakheti" non-UTF-8 character
- Date_C14_Labnr: do the lab numbers really are digit-only? Usually there is some lab identifier (eg. "MAMS-45448" or "Poz-84950")
- Date_BC_AD_Median: for DZN009_ss this median value falls outside the Start-Stop range
- Contamination_Note: I suggest removing the "Estimate and error are weighted means of values per library. Libraries with fewer than 100 were excluded." part as it is not specific to the given individual
- gur017_ES25 and geo015_ES25 individuals are missing all the lab processing information; I realize they come from a different paper, but might be helpful to add if possible; are these indivs a part of another package as well?
- KHT005 (excluded) individual - it would probably be good to change his Group_Name to "GEO_EarlyMiddleAges_excluded" or something (like the "_rel" individuals do); otherwise it might not be picked up as the repeated individual by pipelines (I suppose since it was excluded from the analyses in the original paper, it did not use the provided group name anyway so congruence with the paper should not be an issue here)

nevrome commented:
Thanks for this thorough review, @martynamolak. As always excellent! And thanks for even editing the .bib file directly. 
Some comments:

3. There are no special requirements for `Alternative_IDs` so far. Internal consistency would be good, though.
4. `Poseidon_ID`s are screend for uniqueness on an archive level. So it's fine to reference individuals in other packages of the community-archive. An additional entry in the `Relation_Note` column is not necessary.

Beyond that:

13. List columns should not include spaces between entries, so `BREAD;BREAD` instead of `BREAD; BREAD`.

Please address all 13 points in your revision, @xiaowenjia. I'm happy to help with any additional open questions.

xiaowenjia commented:
Thank you both very much! I will look into them!

xiaowenjia commented:
Hi @nevrome and @martynamolak,
Regarding the points:
1. added myself, thx!
2. Thank you, @martynamolak!
3-6 fixed
7. It seems to like "Samtskhe–Javakheti" to me, is it now all good or should I remove the "-"?
8-10 fixed
11. We filtered the transversion-only site directly on the eigentrat file; the original lab processing information can be found in the package "2023_Koptekin_SouthwestAsia". It might be confusing if we copy them here. I added the filtered snp coverage here to these two samples.
12. KHT005 is not included in this package. I removed the relevant info as well
13. fixed

nevrome commented:
OK - I applied some more minor tweaks to the .janno file. Most notably, the `Source` column should be called `Publication`. I'll merge now.
